i18n/ja.json

```
@@ -28,6 +28,7 @@
     "PROMISE ALL": "Promise.all",
     "ASYNC AWAIT": "async/await",
     "SCOPE": "スコープ",
-    "USE REGEX LUKE": "正規表現を使え、ルーク"
+    "USE REGEX LUKE": "正規表現を使え、ルーク",
+    "FETCH": "fetch API"
```

MDN 的には "Fetch API" のほうが正しいかも
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API

```suggestion
    "FETCH": "Fetch API"
```